src/src/utils/Constants.js

```diff
@@ -3,3 +3,13 @@ export const LOG_IN = "LOG_IN";
 export const LOG_OUT = "LOG_OUT";
 export const SIGN_UP = "SIGN_UP";
 export const USER_TOKEN = "userToken";
+
+// Api
+export const LOG_IN_API = "/api/auth/login";
+export const SIGN_UP_API = "/api/auth/signup";
+export const VIDEOS_API = "/api/videos";
+export const CATEGORIES_API = "/api/categories";
+export const LIKES_API = "/api/user/likes";
+export const WATCHLATER_API = "/api/user/watchlater";
+export const PLAYLIST_API = "/api/user/playlists";
+export const HISTORY_API = "/api/user/history";
```

Comment:
Suggestion you can put all route links this inside one object of routes and export that object

Reply:
Ok will try that